core/src/main/java/pl/agh/edu/engine/hotel/HotelHandler.java

```diff
@@ -22,7 +23,8 @@ public class HotelHandler {
 
 	public final PossibleEmployeeHandler possibleEmployeeHandler = new PossibleEmployeeHandler(this);
 	public final EmployeeHandler employeeHandler = new EmployeeHandler();
-	public final BankAccount bankAccount = new BankAccount(GameDifficultyManager.getInstance().getInitialBalance(), new BigDecimal("0.05"), BigDecimal.valueOf(2));
+	public final BankAccount bankAccount = new BankAccount(GameDifficultyManager.getInstance().getInitialBalance(), new BankAccountDetails(new BigDecimal("0.05"), BigDecimal
```

Comment:
rozumiem że to są te domyślne wartości ale myślę że lepiej by było wziąć info z któregoś banku z configu. Żeby w widoku wyboru było już coś zaznaczone.